src/App/Application.cpp

```diff
@@ -1206,8 +1180,8 @@ std::string Application::getLibraryDir()
 {
 #ifdef LIBRARYDIR
     // #6892: Conda may inject null characters => remove them
-    std::string path = std::string(LIBRARYDIR).c_str();
-    QDir dir(QString::fromStdString(path));
+    auto path = std::string(LIBRARYDIR);
+    const QDir dir(QString::fromStdString(path));

```

Comment:
I think you can use `QStringLiteral` here:
```suggestion
    const QDir dir(QStringLiteral(LIBRARYDIR));
```

Reply:
I just have a bad experience with const with classes that uses the pimpl pattern.
There's no guarantee that the private object doesn't change as it is accessed through a normal pointer and const doesn't propagate. @kadet1090 what's your thoughts on the matter?

You have a good point about keeping the focus of the PR so it is good to avoid the QStringLiteral change :+1: